index.html

```diff
@@ -663,14 +663,23 @@ <h3>Survey of Community UI Developments</h3>
                   <section>
                       <img src="assets/Xi-cam.Acquire/image.png" />
                   </section>
+                  <section>
+                      <p>Finally, various one-off solutions developed by beamline and/or Controls staff at NSLS-II</p>
+                      <ul>
+                          <li>"X-Live" (NSLS-II ISS & QAS)</li>
+                          <li>"xpdacq" (NSLS-II XPD & PDF)</li>
```

Comment:
```suggestion
                          <li>"xpdacq" (NSLS-II XPD & PDF)</li>
                          <li>"XFP High-Throughput Multi-Sample Holder" (NSLS-II XFP)</li>
```

Reply:
No screenshots --- I don't want to dwell on these because I'd rather dwell on how we want things to be built going forward (with queueserver and bluesky-widgets).